### PR TITLE
Removing the word 'master' from the Use and Works ontologies

### PR DESCRIPTION
**Pull request by escowles**

### CODE REVIEW

File: pcdm-ext/use.rdf
Comment (anarchivist) [inline pcdm-ext/use.rdf:44]: Do we have a sense of the use of `pcdmuse:PreservationMasterFile`? Is there a need to indicate that the old class name is deprecated and replaced by `pcdmuse:PreservationFile`?

**Comment by no-reply** [comment]
💯  to this.

a few considerations before moving forward:

  - is "Preservation Master" a term of art we've borrowed from somewhere else? if so, are working bodies addressing this issue there in a way we should consider following?
  - changing URIs is tricky. are we happy to drop the older terms flatly, or is there a deprecation step needed?

**Review by anarchivist — COMMENTED** [review]
I'm all good on the goals of the change, but for the PCDM Use changes, I'm wondering if we want to explicitly deprecate the previous class and indicate it's been replaced by `pcdmuse:PreservationFile`.

**Comment by anarchivist** [comment]
@no-reply 💬 

> changing URIs is tricky. are we happy to drop the older terms flatly, or is there a deprecation step needed?

great minds think alike 🙃

**Comment by escowles** [comment]
@no-reply @anarchivist Good point. A quick look at RDFS and OWL didn't turn up anything obvious, but I'll have time later today or early next week to dig in and see if there's a good way to deprecate the class before removing it outright.

**Comment by no-reply** [comment]
thanks @escowles.

OWL has `owl:deprecated` (with a boolean value). i can't speak to RDFS compatibility, though.

one option to consider is to retain the older term indefinitely with an `owl:sameAs` pointing to the new one.

**Comment by anarchivist** [comment]
it's a bit lighterweight, but this could be a case for `dct:replaces`/`dct:isReplacedBy`. i'd be more concerned about pointing from the old class to new one for PCDM users, rather than reasoning, etc.

**Comment by awoods** [comment]
Even though we are talking about a breaking change, the URL to the current version of the PCDM [use ontology](https://pcdm.org/2015/05/12/use) will still exist. Do we imagine actual application breakage with this change?

**Comment by no-reply** [comment]
wrt "term of art", i found: 
  - https://www.archives.gov/preservation/products/definitions/preservation-copy.html
  - https://library.stanford.edu/research/digitization-services/labs/stanford-media-preservation-lab/capture-specs
  - http://www.digitizationguidelines.gov/term.php?term=archivalmasterfile

the Stanford definitions seem closest to ours, but i think our "best quality" is quite a bit looser than any of these.

it seems to me like dropping "master" has the added benefit of avoiding the implications of these definitions.

**Comment by escowles** [comment]
Thanks for all the feedback on this. I wasn't able to find any good real-world examples, so I'm mostly just going off the comments here and the pretty thin examples in the OWL docs. I've pushed a new commit that is an all-of-the-above approach:
1. Adds the old class linking to the new one with dcterms:isReplacedBy
2. Adds owl:deprecated=true to the old class
3. Adds an rdfs:comment saying the old class is deprecated
4. Updates the new class to link to the old one with dcterms:replaces

I don't expect anyone to be doing reasoning with this ontology, so I would prioritize readability and clarity. If you think any of those four items muddies things more than clarifies, please let me know.

**Review by no-reply — APPROVED** [review]
this looks great to me.

i'll hold off on merging until monday afternoon to give others a chance to chime in.

**Comment by no-reply** [comment]
> Even though we are talking about a breaking change, the URL to the current version of the PCDM use ontology will still exist. Do we imagine actual application breakage with this change?

i think without the deprecation, updating an app to point to load the newer vocabulary version would require a data migration. 
the latest version offers support for a rolling update of existing references. which might lower the bar for adoption of the new term.

**Review by awoods — APPROVED** [review]
This looks good. Thanks @escowles .

We should likely announce the deprecation and timeline for removal of the deprecated term, `PreservationMasterFile`.

**Comment by awoods** [comment]
Per the [PCDM Committers Process](https://github.com/duraspace/pcdm/wiki/PCDM-Committers-Process#voting), I believe we need one more vote of approval, and a wait period of 7 days from today.

In the likely case of a merge, I will plan on publishing the new versions and updating the redirects:
- http://pcdm.org/use# and
- http://pcdm.org/works#

**Review by anarchivist — APPROVED** [review]

**Comment by anarchivist** [comment]
@awoods 👍  from me. I propose that we send a note to the PCDM list about the PR to share information about the change; that message may also include info about a deprecation and proposed timeline to remove.

**Comment by escowles** [comment]
I've posted to the PCDM Google Group that this PR exists and the merging timeline. I proposed leaving the deprecated term in the ontology for a year before removal, given how slow-moving they are.

**Comment by awoods** [comment]
Thanks, @escowles .
For the record, here is a [link to your email message](https://groups.google.com/g/pcdm/c/JOv5TasQyKY/m/cpmCYXjSAAAJ) to PCDM-community.

**Review by mjgiarlo — APPROVED** [review]
Thanks for taking this on and shepherding it through, all.

**Comment by escowles** [comment]
@awoods Can you update the website when you have a chance? Thanks!

**Comment by awoods** [comment]
@escowles : website updated: https://pcdm.org/

**Comment by escowles** [comment]
@awoods Thanks! Sorry to say, I immediately noticed something I didn't update: https://github.com/duraspace/pcdm/pull/74

**Pull request merged**
